### PR TITLE
cluster dashboard + docs

### PR DESCRIPTION
**Pull request by skudasov**

### CODE REVIEW

File: CLUSTER.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Cluster mode
+
+How it works:
```
Comment (sebawo) [inline CLUSTER.md:3]: @skudasov i would like to see here `Motivation` or `Context` section where we're explaining why we even want to need to use this feature.

File: examples/cluster/cluster_test.go [hunk 2]
```diff
@@ -11,15 +11,22 @@ import (
...
-			"env.loki.token":     os.Getenv("LOKI_TOKEN"),
-			"image":              "f4hrenh9it/wasp_test:latest",
-			"jobs":               "3",
-			"sync":               "TestClusterScenario",
-			"env.wasp.log_level": "debug",
+			"env.loki.url":              os.Getenv("LOKI_URL"),
+			"env.loki.token":            os.Getenv("LOKI_TOKEN"),
+			"test.name":                 "TestNodeRPS",
+			"test.timeout":              "24h",
+			"image":                     "f4hrenh9it/wasp_test:latest",
```
Comment (tateexon) [inline examples/cluster/cluster_test.go:23]: Another spot to change to a chainlink ecr.

File: CLUSTER.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+docker build -f Dockerfile.test --build-arg BUILD_ROOT=/go/src/examples/cluster -t wasp_test .
+docker tag wasp_test:latest ${registry}/wasp_test:latest
+docker push ${registry}/wasp_test:latest
+```
+
+- Set [wasp chart](charts/wasp) in test params
+```
+ChartPath: "../../charts/wasp"
+or 
+ChartPath: "oci://registry-1.docker.io/f4hrenh9it/wasp:v0.1.7"
```
Comment (tateexon) [inline CLUSTER.md:31]: And another.

File: CLUSTER.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+```
+- Set `HelmValues` test params
+```
+			"env.loki.url":       os.Getenv("LOKI_URL"),
+			"env.loki.token":     os.Getenv("LOKI_TOKEN"),
+			"test.name":          "TestNodeRPS",
+			"test.timeout":       "24h",
+			"image":              "f4hrenh9it/wasp_test:latest",
```
Comment (tateexon) [inline CLUSTER.md:39]: And another.

**Review by tateexon — CHANGES_REQUESTED** [review]

**Review by sebawo — APPROVED** [review]

**Review by tateexon — APPROVED** [review]

**Pull request merged**